src/osd/OSDMap.cc

```diff
@@ -4238,11 +4238,11 @@ int OSDMap::build_simple_crush_map(CephContext *cct, CrushWrapper& crush,
   ceph_assert(r == 0);
   crush.set_item_name(rootid, "default");
 
+  map<string,string> loc;
+  loc["host"] = "localhost";
+  loc["rack"] = "localrack";
+  loc["root"] = "default";
```

Comment:
```suggestion
  const map<string, string> loc{
    {"host", "localhost"},
    {"rack", "localrack"},
    {"root", "default"}
  };
```

more concise. and for better readability.